COLLABORATOR_GUIDE.md

```diff
@@ -240,14 +240,48 @@ You can find more information [in the full LTS plan](https://github.com/nodejs/l
 
 #### How does LTS work?
 
-Once a stable branch enters LTS, no new features may be added to that release. Changes are
-limited to bug fixes, security updates, possible npm updates, documentation updates, and certain
-performance improvements that can be demonstrated to not break existing applications.
-Semver-minor changes are only permitted if required for bug fixes. Semver-major changes are only
-permitted if required for critical security and bug fixes.
-
-Once a stable branch moves into Maintenance mode, only **critical** bugs, **critical** security fixes,
-and documentation updates will be permitted.
+Once a stable branch enters LTS, no new features may be added to that release.
```

Comment:
Is that true? It seems that it's a bit more subtle than that. See https://github.com/nodejs/node/pull/3609 for instance. It seems that the text below:
> Semver-minor changes are only permitted if required for bug fixes and then only on a case-by-case basis with LTS WG and possibly Core Technical Committee (CTC) review.

 actually describes this subtle situation better, and that that phrase might actually not be needed.